js/views/itemEditVw.js

```diff
@@ -160,23 +160,26 @@ module.exports = Backbone.View.extend({
       contentType: false,
       processData: false,
       data: formData,
-      success: function(data) {
+      success: function(data) { //TODO: Have JQuery parse the JSON directly in ajax call
+        var errorModal, hashArray, imageArray;
         data = JSON.parse(data);
+
         if (data.success === true){
-          var imageArray = __.clone(self.model.get("combinedImagesArray"));
-          var hashArray = __.clone(self.model.get("imageHashesToUpload"));
+          imageArray = __.clone(self.model.get("combinedImagesArray"));
+          hashArray = __.clone(self.model.get("imageHashesToUpload"));
           __.each(data.image_hashes, function (hash) {
-            imageArray.push(self.model.get('server') + "get_image?hash=" + hash);
+            imageArray.push(self.model.get('server') + "get_image?hash=" + hash); //TODO: Change to more formal URL reference from config file
             hashArray.push(hash);
-          })
+          });
           self.model.set("combinedImagesArray", imageArray);
           self.model.set("imageHashesToUpload", hashArray);
 
           self.updateImages();
         }else if (data.success === false){
-          var errorModal = $('.js-messageModal');
+          errorModal = $('.js-messageModal');
```

Comment:
Strict mode is a good practice and it will require variable declaration at the top, for a number of reasons, some of which I've outlined.